chart/values.schema.json

```diff
@@ -687,15 +687,7 @@
             "type": "string",
             "x-docsSection": "Common",
             "default": "CeleryExecutor",
-            "enum": [
-                "LocalExecutor",
-                "LocalKubernetesExecutor",
-                "CeleryExecutor",
-                "KubernetesExecutor",
-                "CeleryKubernetesExecutor",
-                "airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor",
-                "airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor"
-            ]
+            "pattern": "^(LocalExecutor|LocalKubernetesExecutor|CeleryExecutor|KubernetesExecutor|CeleryKubernetesExecutor|airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor|airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor)(,(LocalExecutor|LocalKubernetesExecutor|CeleryExecutor|KubernetesExecutor|CeleryKubernetesExecutor|airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor|airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor))*$"
```

Comment:
I think change affect this:
https://github.com/apache/airflow/blob/main/chart/values.yaml#L658
we need add here if executor contains KubernetesExecutor.

Reply:
Thanks! fixed it

Reply:
Shouldn't we check if executor contains KubernetesExecutor too?

Reply:
If the `workers.keda.query` option is enabled, is it also used when using the `KubernetesExecutor`? I haven't tested it with KEDA yet. (also CeleryKubernetesExecutor..)

Reply:
After this change, we can set executor="CeleryExecutor,KubernetesExecutor". Therefore, if there are tasks running on the KubernetesExecutor, we want to filter out tasks meant for Celery by using a queue filter to decide whether to increase the number of Celery workers.

Reply:
@romsharon98 in [this commit](https://github.com/apache/airflow/commit/3136f22bf7377ed177d7dcca795339c674459913), need to prevent count on k8s queue. come to think, you are right!
I'll make change for now.